### PR TITLE
Duration wrapper padding bug

### PR DESCRIPTION
**Pull request by emilysaffron**
Resolves #NUMBER

**Overall change:** 
Give correct padding on Duration Wrapper for rtl services
Screenshot 2020-12-15 at 09 44 31

**Code changes:**

- css changed so that it works on all browsers

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by jroebu14** [comment]
@emilysaffron code looks good. You'll have to increment the version and update the changelog too. Also, if there is not issue created would it be ok if you added a screenshot to demonstrate the bug? Thanks

**Comment by emilysaffron** [comment]
@jroebu14 thanks, have done this now :)

**Review by pharingee — APPROVED** [review]

File: packages/components/psammead-episode-list/src/Image.jsx [hunk 1]
```diff
@@ -60,9 +60,10 @@ const DurationWrapper = withEpisodeContext(styled.span`
   ${({ service }) => getSansRegular(service)}
   color: ${C_WHITE};
   padding: 0 0 0 ${GEL_SPACING_HLF};
```
Comment (jroebu14) [inline packages/components/psammead-episode-list/src/Image.jsx]: I think you can remove this line @emilysaffron

Comment (emilysaffron) [reply packages/components/psammead-episode-list/src/Image.jsx]: @jroebu14 line 62 ?

Comment (jroebu14) [reply packages/components/psammead-episode-list/src/Image.jsx]: yeh because the next line takes care of that style rule

**Review by jroebu14 — APPROVED** [review]

**Comment by jroebu14** [comment]
Nice work. You'll need to create a Simorgh PR to bring in this new version @emilysaffron

**Pull request merged**